plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/params/PullParams.java

```diff
@@ -25,9 +26,11 @@
  */
 public class PullParams {
 
-    private String image;
-    private String tag;
-    private String registry;
+    private String      image;
+    private String      tag;
+    private String      registry;
+    private String      namespace;
```

Comment:
After last fixes we needn't it any more. In case of docker.io namespace is the same as username.